README.md

```diff
@@ -24,10 +24,6 @@ AlphaPeel is available on [PyPI](https://pypi.org/project/AlphaPeel):
 
     pip install AlphaPeel
 
-## User guide
```

Comment:
@XingerTang was this omission/deletion by accident? I do think we want to point directly to docs - I am aware that PyPI site will be showing that too, but best to give direct link in the README too.

Reply:
@gregorgorjanc That is intended since there is already one user guide link from line 7 to line 9.